politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -531,14 +531,15 @@ func cmdMigrate() error {
 	}
 
 	for i := 0; i < len(users); i++ {
-
 		u := users[i]
 		// Check if username already exists in db. There was a
 		// ~2 month period where a bug allowed for users to be
 		// created with duplicate usernames.
 		_, err = toDB.UserGetByUsername(u.Username)
 
-		paywallIndex = u.PaywallAddressIndex
+		if u.PaywallAddressIndex > paywallIndex {
+			paywallIndex = u.PaywallAddressIndex
+		}
```

Comment:
fixed small bug in logged paywall index in migrate command:
https://github.com/decred/politeia/pull/1419/commits/fae9b3a7fa2a2e27a7117620890c6759a8519399#diff-ab69600787237f42af1cb478e953eb3a838f3b6eca26cd4ea8cafd184345f1b6R540-R542
it used to pick the last user's paywall index always instead of picking the max , it looks better now.

one other thing that i noticed, we don't really store that paywall index to the key_value table in migrate
command, i guess we should do that ?